src/core/Akka/Util/MatchHandler/CachedMatchCompiler.cs

```diff
@@ -71,27 +71,6 @@ private Delegate CompileToDelegate(IReadOnlyList<TypeHandler> handlers, IReadOnl
             delegateArguments = result.Arguments;
             return compiledLambda;
         }
-
-#if !CORECLR
-        /// <summary>
-        /// TBD
-        /// </summary>
-        /// <param name="handlers">TBD</param>
-        /// <param name="capturedArguments">TBD</param>
-        /// <param name="signature">TBD</param>
-        /// <param name="typeBuilder">TBD</param>
-        /// <param name="methodName">TBD</param>
-        /// <param name="methodAttributes">TBD</param>
-        /// <returns>TBD</returns>
-        public void CompileToMethod(IReadOnlyList<TypeHandler> handlers, IReadOnlyList<Argument> capturedArguments, MatchBuilderSignature signature, TypeBuilder typeBuilder, string methodName, MethodAttributes methodAttributes = MethodAttributes.Public | MethodAttributes.Static)
```

Comment:
Byterot, these codes were never compiled before the cleanup (since we moved to single `netstandard2.0` platform target)